doc/source/r_interface.rst

```diff
@@ -46,9 +46,9 @@ of R to pandas objects will be done automatically. For example, to obtain the 'i
 
 .. code-block:: python
 
-    >>> from rpy2.robjects import r  # doctest: +SKIP
-    >>> r.data('iris')  # doctest: +SKIP
-    >>> r['iris'].head()  # doctest: +SKIP
+    >>> from rpy2.robjects import r
```

Comment:
Can you leave all these `doctest: +SKIP`. Or many be it's better to add a `:verbatim:`. In any case, this code should not run if at some point we run doctests of the code in rst files

Reply:
Whoops, I mistook it for a `# noqa` :relaxed: :+1: